strings/base_foundation.h

```diff
@@ -102,9 +102,9 @@ WINRT_EXPORT namespace winrt::Windows::Foundation
 
 namespace winrt::impl
 {
-    template <> inline constexpr auto& name_v<Windows::Foundation::Point> = L"Windows.Foundation.Point";
-    template <> inline constexpr auto& name_v<Windows::Foundation::Size> = L"Windows.Foundation.Size";
-    template <> inline constexpr auto& name_v<Windows::Foundation::Rect> = L"Windows.Foundation.Rect";
+    template <> inline constexpr auto& name_v<Windows::Foundation::Point> = "Windows.Foundation.Point";
```

Comment:
Seems to be a few leftovers here.

Reply:
Perhaps you're looking at an older commit.

Reply:
Hmm, maybe I didn't push that.